src/battleground/BattlegroundContainer.js

```diff
@@ -0,0 +1,51 @@
+//@flow strict
+
+import * as React from 'react';
+import Battleground from './Battleground.js';
+import HealthBarsField from './HealthBarsField.js';
+import Navbar from '../globalComponents/Navbar.js';
+import Tank from '../tanks/Tank.js';
+
+type Props = {||};
+
+type State = {
+	playerOneTank: ?Tank,
+	playerTwoTank: ?Tank,
+};
+
+class BattlegroundContainer extends React.Component<Props, State> {
+
+	render(): React.Node {
+		return (
+			<div className="haveScorebarIfSmall">
+				<Navbar
+					linkName='/BattleArena'
+					returnName='Back to Battle Arena'
+					pageName='Battleground'
+				/>
+				<HealthBarsField
+					playerOneTank = {this.state?.playerOneTank ?? null}
```

Comment:
This.state should never be null. If it is, you have a big problem and want to know about it.

This can just be:
```this.state.playerOneTank```